G1ANT.Addon.Selenium/Api/Services/SeleniumDriverInstaller.cs

```diff
@@ -68,7 +70,14 @@ public void Install(string destinationFolder)
             }
             if (driverToInstall == null)
                 throw new FileNotFoundException($"Cannot find {driverName} driver for browser version {currentBrowserVersion}.");
-            InstallDriverFromRepositoryFolder(driverToInstall.FilePath, destinationFolder);
+
+            if (driverToInstall.Version.Major < currentBrowserVersion.Major && this.BrowserType == BrowserType.Chrome)
```

Comment:
sprawdzenie czy potrzeba zrobic download i sam download powinien byc zrobiiony na poczatku tej metody, jak nie ma plikow w katalogu SeleniumDrivers (a tak sie moze zdarzyc) to nigdy nie nastapi proba pobrania